analyzer/queries/queries.go

```diff
@@ -445,4 +445,97 @@ const (
     JOIN chain.runtime_blocks AS b ON (rt.runtime = b.runtime AND rt.tx_round = b.round)
     WHERE (rt.runtime = $1 AND b.timestamp >= $2::timestamptz AND b.timestamp < $3::timestamptz)
   `
+
+	// Consensus Genesis Queries //
```

Comment:
If we do decide to keep the query strings factored out; I'll probably move this to `/queries/genesis_queries.go`

Reply:
update: we did not pursue this.